### DISCUSSION TITLE
Issue with Vercel Account Signup

### DISCUSSION THREAD

**Discussion by dev-shayan** [vercel/community#4307] [Account Issue] [answered]
### Question

Dear Vercel Support,

I hope this message finds you well. I am writing to you because I encountered an issue while signing up for a Vercel account.

I initially attempted to sign up using my GitHub account but received a message stating that an account is already associated with my GitHub email address. I was advised to log in with my email and password and then link my GitHub account.

However, when I tried to log in using my Gmail account, I received another message stating "We are unable to complete your signup." I have checked my email for any messages from Vercel that might contain more details about the error, but I haven't received any.

I would appreciate any assistance you could provide in resolving this issue. I am eager to start using Vercel and look forward to being able to do so soon.

Thank you for your time and assistance.

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information here in public.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by dev-shayan** [comment]
Dear Amy,

I hope this message finds you well. I wanted to bring to your attention
that I've encountered some difficulties while attempting to sign up for a
new account on Vercel.

I understand that discussing account-specific details in a public forum may
not be secure, and I appreciate your concern for privacy. As advised in
your previous response, I'd like to open a ticket through the appropriate
direct channel to address this matter promptly.

I'd like to emphasize that my case has been registered for the past four
days with the assigned case number #00162816. However, I have not received
any response or update since its registration. As this delay is
significantly affecting my progress in learning NEXT.JS, I kindly request
your urgent attention and assistance in resolving this matter, and I
greatly appreciate your timely response and support.

Thank you for your understanding and assistance.

Best regards,
Shayan

On Wed, 4 Oct 2023 at 02:05, Amy ***@***.***> wrote:

> Sorry you're having trouble signing up for a new account. We can't safely
> discuss account information here in public.
>
> To open a ticket, please use the appropriate direct channel
> <https://vercel.com/guides/how-to-get-vercel-support#direct-channels>
> found in this guide: What is the best way to get support from Vercel?
> <https://vercel.com/guides/how-to-get-vercel-support#direct-channels>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/4307#discussioncomment-7180318>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBGXR4WTINR5LPS5ALMR233X5R4Z3AVCNFSM6AAAAAA5OELIL6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCOBQGMYTQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
